src/index.js

```diff
@@ -1,15 +1,15 @@
 // @ts-check
-const ecstatic = require('ecstatic')
-const http = require('http')
+const LocalWebServer = require('local-web-server')
 const debug = require('debug')('netlify-plugin-cypress')
 const debugVerbose = require('debug')('netlify-plugin-cypress:verbose')
 const { ping } = require('./utils')
 
-function serveFolder (folder, port) {
-  const server = ecstatic({
-    root: folder
-  })
-  return http.createServer(server).listen(port)
+function serveFolder (directory, port, spa) {
+  return LocalWebServer.create({
+    directory,
+    port,
+    spa
+  }).server
```

Comment:
Are there any other `lws` params that could be useful for somebody?
https://github.com/lwsjs/local-web-server